app/grandchallenge/blogs/templates/blogs/post_detail.html

```diff
@@ -20,14 +20,30 @@ <h2>{{ object.title }}</h2>
             {{ object.content|md2html }}
         </div>
         <div class="col-lg-4">
-            <h3>Authors</h3>
-            <ul class="list-group list-group-flush">
-                {% for author in object.authors.all %}
-                    <li class="list-group-item border-0">
-                        {{ author|user_profile_link }}
-                    </li>
-                {% endfor %}
-            </ul>
+            <div class="row mb-1">
+                <h3>Authors</h3>
+            </div>
+            <div class="row mb-3">
+                <ul class="list-group list-group-flush">
+                    {% for author in object.authors.all %}
+                        <li class="list-group-item border-0">
+                            {{ author|user_profile_link }}
+                        </li>
+                    {% endfor %}
+                </ul>
+            </div>
+            <div class="row mb-1">
+                <h3>Tags</h3><br>
```

Comment:
Why is the `<br>` element here? If it is just for spacing then use margins instead for consistency.

We should only show the Tags header if there are tags, right?

Reply:
Good that you mention it, at the moment, tags are required. I should add `blank=True` to the tags field in the post model. 
And the line break can go as well.

Reply:
Blank is only to do with validation, so for any existing objects the m2m through table will be empty. I think that it is fine to keep it as is then, but after we deploy we will need to go back through the old posts and add some tags, I will make a note to do that.

Reply:
oh I was too quick, sorry. I will change it back to what it was, if we really want tags to be required.

Reply:
No it's ok! Let's leave it as is, I don't have strong opinions about it.